var/spack/repos/builtin/packages/exaworks/package.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+class Exaworks(Package):
```

Comment:
Hi @andre-merzky, one change that would be worthwhile is to inherit from `BundlePackage` (like `xsdk`). That handles:

* `has_code = False` (so you can remove that here)
* Also indicates that there are no phases to run when building the package (i.e. sets `phases = []`)

Reply:
Thanks for the feedback, @scheibelp !  The package now inherits from `BundlePackage` as advised :-)